spec.md

```diff
@@ -559,10 +559,52 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+Registries MAY support an enhanced tag listing experience when clients pass an `Accept` header value of 
+`application/vnd.oci.image.index.v1+json`. If a registry supports this mode, they MUST support these additional query parameters:
+* `sort_by` - allowed values are `created_at`, `name`
+* `sort_order` - allowed values are `asc` and `desc`
+
+The response MUST be a json body in the following format:
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 1234,
+      "digest": "sha256:a1a1a1...",
+      "annotations": {
+        "org.opencontainers.image.ref.created": "2022-01-01T14:42:55Z",
+        "org.opencontainers.image.ref.name": "<tag1>"
```

Comment:
Do these annotations originate from the original manifest data, or are they generated by the registry and added to the descriptors? 🤔

- If they come from the manifest data, how does the API handle manifests that do not contain these annotations? 
- If they are generated by the registry, how does the API manage potential conflicts between the manifest annotation values and the registry-recorded values, such as creation time?

Reply:
They are generated by the registry. The registry may optionally include other annotations, including those from the manifest if it chooses.

For conflicts, there _shoudn't_ be any. The `...image.ref.*` tags are specific to a reference, not to a manifest - As in, a manifest shouldn't include an annotation of the reference it will be tagged with _after_ it's pushed. Though maybe there is some weird edge case where this happens :thinking: